pkg/capabilities/consensus/ocr3/ocr3_consensus-schema.json

```diff
@@ -0,0 +1,134 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://github.com/smartcontractkit/chainlink-common/pkg/capabilities/consensus/ocr3/offchain_reporting@1.0.0",
+  "$defs": {
+    "SignedReport": {
+      "type": "object",
+      "properties": {
+        "WorkflowExecutionID": {
+          "type": "string"
+        },
+        "Value": {
+          "type": "object",
+          "properties": {
+            "Underlying": {
+              "type": "object"
+            }
+          },
+          "additionalProperties": false,
+          "required": ["Underlying"]
+        },
+        "Err": {
+          "type": "boolean"
+        }
+      },
+      "additionalProperties": false,
+      "required": ["WorkflowExecutionID", "Value", "Err"]
+    },
+    "FeedValue": {
+      "type": "object",
+      "properties": {
+        "deviation": {
+          "type": "string",
+          "description": "The deviation that is required to generate a new report. Expressed as a percentage. For example, 0.01 is 1% deviation."
+        },
+        "heartbeat": {
+          "type": "integer",
+          "minimum": 1,
+          "description": "The interval in seconds after which a new report is generated, regardless of whether any deviations have occurred. New reports reset the timer."
+        },
+        "remappedID": {
+          "type": ["string", "null"],
+          "description": "An optional remapped ID for the feed."
+        }
+      },
+      "additionalProperties": false,
+      "required": ["deviation", "heartbeat"]
+    }
+  },
+  "properties": {
+    "config": {
+      "properties": {
+        "aggregation_method": {
+          "type": "string",
+          "enum": ["data_feeds"]
+        },
+        "aggregation_config": {
+          "type": "object",
+          "properties": {
+            "allowedPartialStaleness": {
+              "type": "string",
+              "description": "Allowed partial staleness as a number between 0 and 1."
+            },
+            "feeds": {
```

Comment:
This is specific to the "data_feeds" aggregator and won't work with other ones. Why do you want to put it in the schema?

Reply:
Yeah, this is meant to represent the capability as it is today.  I'll add support for having different methods in a follow-up PR.